server/controllers/CommentController.js

```diff
@@ -36,6 +40,34 @@ class CommentController {
         User.findByPk(userId)
           .then((user) => {
             const { fullName, avatarUrl } = user;
+            ArticleLike.findAll({
+              where: {
+                articleId,
+                status: true
+              },
+              include: [{ model: User }]
+            })
+              .then((users) => {
+                // looping through the list of users who liked the article
+                users.map((userObj) => {
+                  sendEmail(userObj.User, {
+                    subject: 'New Article Reaction',
+                    body: `You have new article reaction. 
+                    Check it out at 
+                    ${process.env.API_BASE_URL}/article/${articleId}`
+                  });
+                  // add a new entry to notification events
+                  NotificationController.addNewNotificationEvent(
+                    'article reaction',
+                    userObj.User.id, user.id,
+                    'You have a new reaction on an article you liked',
```

Comment:
I think this should say 'you have a new comment on an article you liked' because it is more descriptive.

Reply:
You are right. Thanks

Reply:
Feedback implemented @KvNGCzA

Reply:
Feedback implemented @augustineezinwa